src/main/java/Sunday.java

```diff
@@ -0,0 +1,33 @@
+import command.Command;
+import exceptions.SundayException;
+import utilities.Parser;
+import utilities.Ui;
+
+public class Sunday {
+    private Ui ui;
+    public Sunday(String filepath) {
+        this.ui = new Ui();
+        try {
+            Command.INITIALIZE.execute("data/sunday.txt");
+        } catch (SundayException e) {
+            Ui.printException(e);
+        }
+    }
+
+    private void run() {
+        Command command = null;
+        while (command != Command.BYE) {
+            try {
+                String[] fullCommand = ui.readCommand();
+                command = Parser.parse(fullCommand);
+                command.execute(fullCommand[1]);
+            } catch (SundayException e) {
+                Ui.printException(e);
+            }
+        }
+        this.ui.close();
```

Comment:
Do you think having another method for destroying the ui will make it easier to understand? 🤔

Reply:
This method was created to close the Scanner object that took inputs from the user. Apologies for the confusion!